program/plugins/nikto_msgs.plugin

```diff
@@ -66,10 +66,10 @@ sub nikto_msgs {
         	$name =~ s#(^.*<ComputerName>)([^<]+)(</ComputerName>.*$)#$2#;
         	my $eposerver = $content;
         	$eposerver =~ s#(^.*<ePOServerName>)([^<]+)(</ePOServerName>.*$)#$2#;
-        	add_vulnerability( $mark, "Web server is a McAfee ePO agent and showing the hostname as '$name' and the ePO server(s) as '$eposerver'.", 980100, "https://www.mcafee.com/enterprise/en-us/downloads/trials/epo-mcafee-agent-deployment.html", "GET", "/_LOGFILENAME_", $request, $resonse);
+        	add_vulnerability( $mark, "Web server is a McAfee ePO agent and showing the hostname as '$name' and the ePO server(s) as '$eposerver'.", 980100, "https://www.mcafee.com/enterprise/en-us/downloads/trials/epo-mcafee-agent-deployment.html", "GET", "/_LOGFILENAME_", $request, $response);
```

Comment:
This looks like an actual programming error, a typo in a variable name, not just a typo in user-visible text or the documentation.